src/app/faq/faq.component.scss

```diff
@@ -0,0 +1,99 @@
+@import 'src/assets/styles/variables';
+
+.global-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.faq-container {
+  @extend .global-column;
+  background-image: url('/assets/images/faq-bg.png');
+  background-repeat: no-repeat;
+  height: 100%;
+  width: 100%;
+  &__header {
+    display: flex;
+    flex-direction: row;
+    margin-top: 48px;
+    &-title {
+      @extend .global-column;
+      margin-left: 26%;
+      @media screen and (max-width: 600px) {
+        margin-left: 10px;
+      }
+      &-icon {
+        color: $cerise;
+        margin: 8% 0 0 32%;
```

Comment:
we normally don't use percentage as a unit.

Reply:
resolved.....................Thank you